src/main/java/duke/tasklist/TaskListStub.java

```diff
@@ -0,0 +1,20 @@
+package duke.tasklist;
+
+/** Represents a stub for TaskList to remove dependencies during unit testing. */
+public class TaskListStub extends TaskList {
+
+    public TaskListStub() {
+    }
+
+
+    /**
+     * Return the value 3.
```

Comment:
Should be `Returns` instead of `Return`.

Reply:
You are right!